src/org/omegat/gui/editor/autocompleter/AutoCompleter.java

```diff
@@ -93,6 +94,13 @@ public class AutoCompleter implements IAutoCompleter {
     public AutoCompleter(EditorTextArea3 editor) {
         this.editor = editor;
 
+        popupTimer = new Timer(100, e -> {
```

Comment:
It is better to define constant with a value "100" and replace it with constant.